contents/tutorials/nextjs-analytics.md

```diff
@@ -199,42 +200,44 @@ export default NextAuth({
 
-![GitHub](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/tutorials/nextjs-analytics/github.png)
+![GitHub](https://res.cloudinary.com/dmukukwp6/image/upload/w_1600,c_limit,q_auto,f_auto/Clean_Shot_2025_08_13_at_10_08_00_2x_d38cb2039c.png)
 
-With these, create a `.env.local` file in the main app (`tutorial`) folder and include the `NEXTAUTH_URL` ([`http://localhost:3000/`](http://localhost:3000/) for now) and a `NEXTAUTH_SECRET` (which you can generate on [this site](https://generate-secret.vercel.app/32) or by creating a random 32-character string).
+Copy the client ID and generate and copy a new client secret. With these, create a `.env.local` file in the main app (`pages-tutorial`) folder and set them as `GITHUB_ID` and `GITHUB_SECRET`. You'll also need to add a `NEXTAUTH_URL` ([`http://localhost:3000`](http://localhost:3000/) for now) and `NEXTAUTH_SECRET` (which you can generate on [this site](https://generate-secret.vercel.app/32) or by creating a random 32-character string) value.
 
 ```bash
 GITHUB_ID=<github_client_id>
 GITHUB_SECRET=<github_client_secret>
-
 NEXTAUTH_URL=http://localhost:3000
 NEXTAUTH_SECRET=<random_32_character_string>
 ```
 
 ### Setting up sessions
 
-With NextAuth and GitHub set up, we now have the infrastructure to authenticate users. Now we can implement user sessions to let them log in and out as well as get their details. The first step to doing this is adding a `SessionProvider` from `next-auth` to `_app.js` like this:
+With NextAuth and GitHub set up, we now have the infrastructure to authenticate users. Now, we can implement user sessions to let them log in and out as well as get their details. 
+
+The first step to doing this is adding a `SessionProvider` from `next-auth/react` to `_app.js` like this:
 
 ```js
 // pages/_app.js
+import "@/styles/globals.css";
 import { SessionProvider } from "next-auth/react"
 
-export default function App(
-  { Component, pageProps: { session, ...pageProps } }
-) {
+export default function App({ Component, pageProps: { session, ...pageProps } }) {
   return (
     <SessionProvider session={session}>
       <Component {...pageProps} />
     </SessionProvider>
-  )
+  );
 }
 ```
 
-Next, add the session details, and the ability to sign in and out to our `index.js` page. We can do this with the `next-auth` `useSession()` hook. `next-auth` also provides methods for signing in and out. 
+Next, add the session details, and the ability to sign in and out to our `index.js` page. We can do this with the NextAuth `useSession()` hook as well as its methods for signing in and out.
 
-If there is a session, show details about the user and a button to sign out. If there isn’t a session, show a button to sign in. Together, it looks like this:
+We set up a check to see if there is a session and show details about the user and a button to sign out. If there isn’t a session, we show a button to sign in. Together, it looks like this:
 
 ```js
```

Comment:
I think this is editing an existing file. Consider using the `// +` comment trailing a line to highlight lines added here. Might make it easier to follow in longer tutorials like this.